portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +24,11 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private String welcome = "Welcome to my portfolio!";
```

Comment:
nit: Since this is a constant class variable, you can define it as `private static final String WELCOME` (stylistically constants are typically in all-caps in Java)